package.json

```diff
@@ -71,6 +71,7 @@
     "@react-native/typescript-config": "0.73.1",
     "@types/react": "^18.2.6",
     "@types/react-test-renderer": "^18.0.0",
+    "@types/react-native": "0.73.0",
```

Comment:
This dependency [appears to be deprecated](https://www.npmjs.com/package/@types/react-native).   Also I noticed that `0.73.0` is the last version, do you know if this is still an issue in React Native `0.74`?

Reply:
Yes, the dependency is marked as deprecated, but adding this seems to be the only solution which fixes this issue [#376]

I tried out upgrading react-native version to `0.74.3` and I get the same error.